.github/scripts/downstream-project-spl-common.sh

```diff
@@ -27,3 +27,11 @@ fi
 
```

Comment:
```suggestion

echo "[profile.dev.package.curve25519-dalek]
opt-level = 3" >> Cargo.toml

```
The slow-down from the upgrade can be avoided by bumping up the developer opt-level to 3 (https://github.com/anza-xyz/agave/pull/2683).